devnotes.md

```diff
@@ -49,13 +72,48 @@ For me, this was:
 
 ```git clone https://github.com/kostyat/inferelator_ng.git```
 
-#4. Running the unit tests
+## Set up the "remote" repository (once)
+
+You will want to periodically 
+[merge changes from the main repository into your fork](https://help.github.com/articles/syncing-a-fork).
+This does not happen automatically -- to make it happen you need to 
+[set up the main repository as a "remote"](https://help.github.com/articles/configuring-a-remote-for-a-fork)
+
+`git remote add upstream https://github.com/simonsfoundation/inferelator_ng.git`
+
+## Merging changes from the remote into your fork (as needed)
+
+To merge changes from the main remote repository into your fork 
+[github suggests](https://help.github.com/articles/syncing-a-fork) the following
+sequence
+
+```
+# Fetch the upstream changes
```

Comment:
I changed the procedure for pulling from the remote to match the work flow documented in the github documentation.